src/pages/editor/mind/components/EditorToolbar/KoniToolbar.tsx

```diff
@@ -0,0 +1,3 @@
+import FlowToolbar from './FlowToolbar';
+
+export default FlowToolbar;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of default exports is forbidden](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)